### PR TITLE
i77 Generate PDF of formal spec

### PR DESCRIPTION
**Pull request by kerimoyle**
Addresses #77 

This PR generates a single PDF which represents only the formal text of the normative specification, and does not include any of the informative spec blocks, where the text and heading numbers in the PDF are identical to those in the HTML.

- [x] Figure out where to put the preamble stuff ...
- [x] Get the interpunct character to render in the Built-in units table PDF version, see #87 
- [ ] Get preamble address correct?  Do we need a DOI? #104
- [ ] Find a Latex wizard to help with formatting once the content (all other PRs) are finalised.

Rendered version is here: https://cellml-specification-dev.readthedocs.io/en/i77_pdf_of_formal_spec_only/index.html

### CODE REVIEW

**Comment by MichaelClerx** [comment]
I'm trying to try this out but running into trouble getting this branch on my machine locally.

I've added your repo to my git, but then what...?

**Comment by kerimoyle** [comment]
If you've forked my repo (kerimoyle/cellml-specification), you should be able to pull and push from your own fork as normal?  If you've already forked _this_ repo (cellml/cellml-specification) then you should be able to see this PR as a branch you can refer to? Perhaps referred to as something like "pr/95" or by its name?

**Comment by nickerso** [comment]
> but then what...?

We work very similar to libCellML. Take a look at 
https://www.libcellml.org/en/latest/dev_setup.html#forking-your-own-copy - you'd need to add a remote to @kerimoyle's repo and fetch the branch from there...I don't think the pull request exists as a branch in this repo (cellml/cellml-specification).

**Comment by kerimoyle** [comment]
No changes to content, only getting the PDF to render nice(r)!  I'd like to get this one in so that I can start on addressing #110 without too many conflicts ... There will be more to come on the PDF front, but I'll do that after the headings have changed so we can better see what it looks like.  So ... ok to merge? @agarny @nickerso @hsorby ?  Here's the rendered PDF at present:
 https://cellml-specification-dev.readthedocs.io/_/downloads/en/i77_pdf_of_formal_spec_only/pdf/

File: reference/informative_preamble.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+and Maxwell L. Neal.
+
+The authors also wish to acknowledge the significant contribution of the (discontinued) draft CellML 1.2 specification, much of the text of which was incorporated into this specification, although the semantics have changed considerably.
+The CellML 1.2 specification was itself the result of a collaborative effort by a number of researchers during 2008-2011\:
+
+Andrew K. Miller (who wrote the text reused here),
+Randall Britten,
+Jonathan Cooper,
+Alan Garny,
+Peter J. Hunter
```
Comment (hsorby) [inline reference/informative_preamble.rst]: missing comma

File: formal.rst [hunk 2]
```diff
@@ -1,18 +1,53 @@
...
 
+**Authors:**
+Michael Clerx,
+Michael T. Cooling,
+Jonathan Cooper,
+Alan Garny,
+Keri Moyle,
+David P. Nickerson,
+Poul Nielsen,
+and Hugh Sorby.
```
Comment (agarny) [inline formal.rst]: Shouldn't the "and" be on the previous line? (Same below.)

File: index.rst [hunk 3]
```diff
@@ -1,18 +1,16 @@
 .. _index:
 
 ==========================================
-Normative specification for CellML 2.0
+CellML 2.0 Specification
 ==========================================
```
Comment (agarny) [inline index.rst]: I would remove some "=" to make it the same length as the header.

Comment (agarny) [reply index.rst]: Same below.

Comment (nickerso) [reply index.rst]: or leave it...I was planning to put the normative spec bit back in once things settle down ;-)

File: reference/informative_preamble.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.. _inform_preamble:
```
Comment (agarny) [inline reference/informative_preamble.rst]: No way to have some kind of include file? It looks like it overlaps quite a bit (entirely) with `formal.rst`.

**Comment by kerimoyle** [comment]
@nickerso It is there twice, because the PDF renderer doesn't recognise the interpunt symbol, so they're a little different between the `..only:: html` and `..only:: latex` versions.

File: reference/informative_preamble.rst (hunk 4 above)
Comment (kerimoyle) [reply reference/informative_preamble.rst]: True, but there are differences in the order of where things will appear, and eventually they will be more distinct as we get more information about the informative spec in the inform_preamble file.  I've found that using includes does odd stuff to the numbering and structure of the PDF, so need to keep that side as is for now.

File: formal.rst (hunk 2 above)
Comment (kerimoyle) [reply formal.rst]: It's all rendered to a single line anyway ...

File: index.rst (hunk 3 above)
Comment (kerimoyle) [reply index.rst]: Will do elsewhere throughout later on

File: formal.rst (hunk 2 above)
Comment (kerimoyle) [reply formal.rst]: 034c2ef done

File: reference/informative_preamble.rst (hunk 1 above)
Comment (kerimoyle) [reply reference/informative_preamble.rst]: 034c2ef done

**Comment by agarny** [comment]
> @nickerso It is there twice, because the PDF renderer doesn't recognise the interpunt symbol, so they're a little different between the `..only:: html` and `..only:: latex` versions.
> 
> Ah, dang, you're right ... it's there twice in the PDF ... oops

This aside, can't we find a way around the interpunt symbol? I really don't like the idea of having that table twice (not good when it comes to maintenance).

**Comment by kerimoyle** [comment]
@agarny Please do!  I've spent a day trying and failing ... :\  (hence most of the massive commit history of this branch!!).

**Comment by kerimoyle** [comment]
It's also now defaulting to a `.` in both tables, which is something new ... previously it was simply omitted.  If that behaviour is stable then we can just remove the second table ...

**Comment by kerimoyle** [comment]
OK, double table is fixed, new pages added in the PDF before both prefix and built-in units tables.  Only outstanding thing here are general formatting (length of title decoration) which will be done in another PR, and the interpunt thing (which will be done when pigs fly).  I'm going to merge this now and will add the rest as needed. ...

**Comment by MichaelClerx** [comment]
Awesome!

**Pull request merged**